app/forms/journeys/early_years_payment/practitioner/check_your_answers_form.rb

```diff
@@ -7,6 +7,10 @@ class CheckYourAnswersForm < Form
         def save
           return false if invalid?
 
+          if journey.requires_student_loan_details?
+            journey::AnswersStudentLoansDetailsUpdater.call(journey_session)
+          end
```

Comment:
Not one for this PR but makes sense to do this for all journeys, probably for when we split personal details for the other journeys